### ISSUE TITLE
cuDNN failed to initialize

### ISSUE THREAD

**Issue by xiaoleiwang10** [CSBDeep/CSBDeep#36]
Dear everyone,
When I test the example about probilistic neural networking and run "restored = model.predict(x, axes, normalizer=None)", I got error. The error said: (0) Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above. [[{{node down_level_0_no_0_1/convolution}}]].
My software versions are:
Cuda10.1
cudnn7.6.3.30
Tensorflow-gpu:the latest version.

Do you know what's the problem? Any information would be helpful!

Thank you,
Xiaolei

**Comment by uschmidt83** [comment]
Hi Xiaolei,

I don't know what the reason for your problem is.
However, I've had similar errors appear simply because I had another process/notebook running that used all of the GPU memory.

Sorry for the late reply.

Best,
Uwe

**Comment by maweigert** [comment]
Hi @xiaoleiwang10,

In case you used conda to install tensorflow, this is most likely due to the wrong cuda/cudnn combination installed by it. I would recommend trying to reinstall tensorflow/cuda/cudnn with the following steps

1. Check your nvidia driver version (via nvidia-smi, e.g. `387.34`)
2. See what CUDA/cudnn version it maximally supported by your driver:
https://docs.nvidia.com/deeplearning/sdk/cudnn-support-matrix/index.html#cudnn-cuda-hardware-versions (e.g. `Cuda 9.0` for the driver above)
3. See what tensorflow 1.x version is maximally supported:
https://stackoverflow.com/questions/50622525/which-tensorflow-and-cuda-version-combinations-are-compatible (e.g. `1.12` for the cuda version above)
4. Install all packages with the correct versions manually e.g. (for the driver/cuda/tf versions above)
`conda install cudatoolkit=9.0 cudnn=7.6.4 tensorflow-gpu=1.12`

**Comment by maweigert** [comment]
Closing due to non-activity (please reopen if still relevant)
